chef/cookbooks/ceph/providers/pool.rb

```diff
@@ -0,0 +1,23 @@
+def whyrun_supported?
+  true
```

Comment:
Are you sure? 
According to http://dougireton.com/blog/2013/01/07/creating-an-lwrp-part-2/ make a resource supporitng ```whyrun``` requires adding some ```converge_by``` blocks. I don't see any. So maybe you just want to remove this.